Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs

```diff
@@ -123,7 +111,7 @@ public static bool CheckFriendlyFirePlayerRules(ReferenceHub attackerHub, Refere
                         }
                     }
                 }
-                else if(!attacker.UniqueRole.Equals(string.Empty))
+                else if(!string.IsNullOrEmpty(attacker.UniqueRole))
```

Comment:
This should never be null unless a developer screwed up but sure - this works as well

Reply:
You're right, but `IsNullOrEmpty` makes more sense here then `Equals`, and an extra check for `null` wouldn't hurt